README.md

```diff
@@ -27,6 +27,10 @@ What is pkgbase?
 FreeBSD has had the `pkg` system for years now, giving users easy installation and management of third party applications from the ports tree. "pkgbase" refers to the process of getting the FreeBSD base system (userland and kernel) into package form, allowing the same management tools to be used. This ends the requirement of multiple tools to update a system, like `freebsd-update` and then `pkg`. Additionally, it more tightly integrates the standard FreeBSD base system and the ports/package repos. Now updates are done consistently across the two, ensuring packages and userland don't run into problems that result from a mis-matched userland/kernel and third party applications.
 
 
+What is a CFT?
```

Comment:
Before explaining what a CFT is, the document should mention a CFT.  But it does not.  Explaining the term to the FreeBSD community is probably not necessary at all, and maybe even insulting.

If you want to talk about a CFT, just word it as "Call for Testing (CFT)".  The community either knows what it means or knows where to find out.

Reply:
We have had a lot of questions what a CFT is on Mailing list and Reddit.